firebase/firebase.json

```diff
@@ -7,6 +7,26 @@
     "predeploy": [
       "npm --prefix \"$RESOURCE_DIR\" run lint",
       "npm --prefix \"$RESOURCE_DIR\" run build"
+    ],
+    "source": "functions"
+  },
+  "hosting": {
+    "public": "public",
+    "ignore": [
+      "firebase.json",
+      "**/.*",
+      "**/node_modules/**",
+      "us/**"
+    ],
+    "rewrites": [
+      {
+        "source": "/share/**",
+        "function": "dynamicWebContent"
```

Comment:
This directs all /share/... requests to the "dynamicWebContent" firebase function.

In the future we could serve other requests as well (e.g. to do the meta tag replacement stuff.)